aws-iot-softwarepackage/.rpdk-config

```diff
@@ -0,0 +1,26 @@
+{
+    "artifact_type": "RESOURCE",
+    "typeName": "AWS::IoT::SoftwarePackage",
+    "language": "java",
+    "runtime": "java8",
+    "entrypoint": "software.amazon.iot.softwarepackage.HandlerWrapper::handleRequest",
+    "testEntrypoint": "software.amazon.iot.softwarepackage.HandlerWrapper::testEntrypoint",
+    "settings": {
+        "version": false,
+        "subparser_name": null,
+        "verbose": 0,
+        "force": false,
+        "type_name": null,
+        "artifact_type": null,
+        "namespace": [
+            "software",
+            "amazon",
+            "iot",
+            "softwarepackage"
+        ],
+        "codegen_template_path": "default",
+        "protocolVersion": "2.0.0"
+    },
+    "logProcessorEnabled": "true",
```

Comment:
What's the point of this line?
"logProcessorEnabled": "true", looks for other iot resources it doesn't have this line  https://tiny.amazon.com/16ko2ptoa